build/styleguide.config.js

```diff
@@ -15,8 +15,10 @@ module.exports = {
     require('./webpack.prod.config.js'),{
       devServer: {
 	contentBase: resolve('../docs'),
-	publicPath: '/docs/',
-	disableHostCheck: true
+	publicPath: '/dev/',
+	disableHostCheck: true,
+	openPage: '/dev/',
```

Comment:
So this allows us to run prod and dev version of  Styleguide at the same time:

prod version is on `/`
dev version is on `/dev/`